src/configuration/services/redis.md

```diff
@@ -24,7 +24,7 @@ The following versions are available but are not receiving security updates from
 
 ### Ephemeral Redis
 
-The `redis` service type is configured to serve as a LRU cache with the eviction policy `allkeys-lru` - its storage is not persistent.  It is not suitable for use except as a disposable cache.
+The `redis` service type is configured to serve as a LRU cache; its storage is not persistent.  It is not suitable for use except as a disposable cache.
 
 To add an Ephemeral Redis service, specify it in your `.platform/services.yaml` file like so:
```

Comment:
Since these two sections are separated, readers might not immediately know this is something that could be configured. Maybe we should link to that section in the next paragraph?

"Data in an Ephemeral Redis instance is stored only in memory, and thus requires no disk space. When the service hits its memory limit it will automatically evict old cache items by default to make room for new ones, but it is also possible to [configure a different eviction policy](/configuration/services/redis.md#setting-the-eviction-policy)."

Reply:
As per your comment, I don't think it's necessary to add this as a subsection to Ephemeral Redis, so long as you link to it like I suggested.